src/libs/TransactionUtils/index.ts

```diff
@@ -640,6 +640,10 @@ function getUpdatedTransaction({
     }
 
     if (Object.hasOwn(transactionChanges, 'attendees')) {
+        updatedTransaction.comment = {
+            ...updatedTransaction.comment,
+            attendees: transactionChanges.attendees,
+        };
```

Comment:
Why did we decide to update this?

Reply:
It fixes this scenario:

Steps:
1. You create an expense add 2 attendee.
2. Go offline.
3. Edit the attendee on created expense.
4. Try to edit again.
5. Check the selected users on the attendee page.

This method (getUpdatedTransaction) receives `transactionChanges` as one of the arguments. When `comment.attendees` are changed on the transaction this method receives `attendees` (via param mentioned before) that are not nested inside the `comment` and updates `modifiedAttendees` instead.

This method is only fired upon editing expense and so far I wasn't able to find anything broken after this change